docs/content/guides/developer/getting-started/graphql-rpc.mdx

```diff
@@ -333,6 +333,39 @@ fragment DynamicFieldValueSelection on DynamicFieldValue {
   }
 }
 ```
+### Execute a transaction
+
+Transaction execution takes in two arguments, `txBytes` and `signatures`. `txBytes` is the serialized unsigned transaction data, which can be generated using Sui CLI's `--serialize-unsigned-transaction` option. The `signatures` can be generated using Sui CLI's keytool command `sui keytool sign`. More information on Sui CLI can be found [here](/references/cli).
```

Comment:
Is this the right way to reference another doc? @stefan-mysten

Reply:
Yes, that looks good!

Reply:
```suggestion
Transaction execution takes in two arguments, `txBytes` and `signatures`. `txBytes` is the serialized unsigned transaction data, which can be generated when using the Sui CLI's `client call` [command](/references/cli/client), to call a Move function by passing the `--serialize-unsigned-transaction` flag. The `signatures` can be generated using Sui CLI's [keytool](/references/cli/keytool) command `sui keytool sign`. More information on Sui CLI can be found [here](/references/cli).
```